packages/agreement-process/.env

```diff
@@ -24,8 +24,10 @@ S3_BUCKET=interop-local-bucket
 S3_CUSTOM_SERVER=true
 S3_SERVER_HOST=http://localhost
 S3_SERVER_PORT=9000
+MINIO_ACCESS_KEY=test-aws-key
+MINIO_SECRET_KEY=test-aws-secret
```

Comment:
if I'm not missing something, these credentials are not required if we use `aws.config.local` file (same for others `.env` files)
```suggestion
```